src/desktop/chat/chatlineeditandroid.cpp

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Why create a whole separate TU for this? The idiomatic way to handle message send on every Android messaging app I have ever used (and some desktop apps too) with multi-line chat is to have a send button; can we just add one of those? (It would be a good idea to have a send button in any case since it is conventional.)